operations/configuration_management/Worker_Image_Customization.md

```diff
@@ -37,8 +75,18 @@ The following procedure describes how to correctly edit the `bootprep` files to
 
     Perform the steps in [Management Node Image Customization](Management_Node_Image_Customization.md), with the following notes:
 
+    - The linked procedure gives examples of customizing a Kubernetes NCN image, which is what needs to be done in this case.
+    - In the steps to identify the NCN image and obtain its artifacts, do the following based on the current scenario:
+      - (`ncn-m001#`) If doing this as part of a CSM upgrade, then use the following command to set the `ARTIFACT_VERSION` variable.
+
+          ```bash
+          ARTIFACT_VERSION=$(grep "^export KUBERNETES_VERSION=" /etc/cray/upgrade/csm/myenv | tail -1 | cut -d= -f2)
```

Comment:
Also not really a huge issue but I kind of feel like sourcing the file would be cleaner, i.e.

```
source /etc/cray/upgrade/csm/myenv
ARTIFACT_VERSION=$KUBERNETES_VERSION
```

I have no idea if there would be unintended consequences to that though since I don't know what else is in `myenv`.

Reply:
Yeah, that is why I didn't do it that way. I didn't want to create that open-ended of a dependency here, since this isn't really the purpose of that file.